config/crd/bases/solr.apache.org_solrbackups.yaml

```diff
@@ -1053,9 +1053,12 @@ spec:
                 type: object
               repositoryName:
                 description: The name of the repository to use for the backup.  Defaults to "legacy_local_repository" if not specified (the auto-configured repository for legacy singleton volumes).
+                pattern: '[a-zA-Z0-9]([-_a-zA-Z0-9]*[a-zA-Z0-9])?'
                 type: string
               solrCloud:
                 description: A reference to the SolrCloud to create a backup for
+                minLength: 63
```

Comment:
[-1] I imagine you put this in for testing at some point or something?

Reply:
nah, I actually meant maxLength here (because kubernetes resource names cannot go beyond 63 characters: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names)

Good find!